registry/src/skills.json

```
@@ -373,5 +373,37 @@
     "minimumSkillBalance": 0,
     "status": "ACTIVE",
     "isDefault": true
+  }, 
+  {
+    "pluginId": "79449cd9-0590-4ccc-9d9f-12f99438c17b",
+    "name": "@moxie-protocol/plugin-degenfans-alfafrens",
+    "displayName": "DegenFans AlfaFrens",
+    "version": "0.1.8+build.1",
```

Make this consistent w/ `package.json`
```suggestion
    "version": "0.0.1",
```